src/types.ts

```diff
@@ -41,14 +41,12 @@ export type BeforeLanguageSwitchHandler = <Context = unknown>(
   newLocale: string,
   initialSetup: boolean,
   context: Context
-) => string | void
+) => Promise<unknown>
```

Comment:
`BeforeLanguageSwitchHandler`  should be removed, so because it's swtiched to nuxt hooks, 
It is no longer necessary to configure in nuxt.config.ts.